research/src/pages/select/select.proposal.mdx

```diff
@@ -117,6 +117,30 @@ children:
   Should this apply to the entire anatomy or solely the &lt;option&gt; and &lt;optgroup&gt; element
 </p>
 
+### Accessibility Implementation <a class="link" id="assistive-tech"></a>
+
+The `<select>` element represents a "combobox" and would be mapped as such across the different accessibility APIs per the [`combobox` role](https://www.w3.org/TR/core-aam-1.1/#role-map-combobox) as defined by CORE AAM and the [HTML AAM spec](https://www.w3.org/TR/html-aam-1.0/#html-element-role-mappings). The `<listbox>` popup would be mapped per the [`listbox` role](https://www.w3.org/TR/core-aam-1.2/) as defined by CORE AAM. **Note:** awaiting clarity on which `listbox` mapping to use, as one may be specific to the deprecated ARIA 1.1 `combobox` pattern. 
```

Comment:
I'm wondering if we should also try to fit a link to https://w3c.github.io/aria/#combobox in here somewhere.
Clicking the `combobox` link in the left column at https://www.w3.org/TR/core-aam-1.1/#role-map-combobox leads to https://www.w3.org/TR/wai-aria-1.1/#combobox, which suggests that the combobox role is strictly used with textboxes, as opposed to the newer guidance in 1.2 that it can also be used with a button.

Reply:
yeh.  can definitely add those.  good call

Reply:
links have been updated.  we should be adhering to the latest definition of `combobox`.  The ARIA 1.1 definition is not a viable path forward as it is deprecated.